Podfile.lock

```diff
@@ -71,7 +71,7 @@ PODS:
     - GZIP (~> 1.2.2)
     - libbase58 (~> 0.1.4)
     - OLMKit (~> 3.0.0)
-    - Realm (~> 3.13.1)
+    - Realm (~> 3.11.1)
```

Comment:
Why did realm get downgraded?

Reply:
To make pod install working with MatrixKit 0.9.5

Reply:
riot-ios/develop should use pods of kit/develop & sdk/develop

Reply:
If you uncomment `$matrixKitVersion = 'develop'` and perform a pod install it will update the podfile.lock correctly and bump Realm to 3.13.1. The podfile.lock here was generated by using default kit version `$matrixKitVersion = '0.9.5'` in podfile.

Reply:
> If you uncomment `$matrixKitVersion = 'develop'` and perform a pod install it will update the podfile.lock correctly and bump Realm to 3.13.1.

yes. This is what I meant to avoid to do a stupid `rm Podfile.lock` the day of the release.